scalableminds/webknossos
# Give some love to public tracings

Disable certain UI elements and hide permission error message for public tracings.

### Mailable description of changes:
 - Disabled some UI elements for public tracings, that would've resulted in permission errors on use.

### Steps to test:
- Share a tracing and make it publicly accessible. Access it when logged out (disable `enableDevAutoLogin` in `application.conf`). There should be no error message, the tracing should not be re-nameable, the share modal should not include the checkbox to make the tracing private again and the actions dropdown next to the read-only tag should only contain actions that make sense in the public tracing context.
- Nothing should've changed when logged in, except that read-only tracings should not be re-nameable now, which was a bug before.

### Issues:
- fixes #1997

------
